Sources/MapboxSearch/InternalAPI/CoreAliases.swift

```diff
@@ -1,10 +1,8 @@
-@_implementationOnly import MapboxCommon_Private
+import MapboxCommon_Private
+import MapboxCoreSearch
+import MapboxCoreSearch_Private
 
-/// https://forums.swift.org/t/update-on-implementation-only-imports/26996
-@_implementationOnly import MapboxCoreSearch
```

Comment:
Makes me think, `@ _implementationOnly` isn't the only experimental attribute we use, there is also `@_exported`: https://github.com/search?q=repo:mapbox/mapbox-search-ios%20_exported&type=code

Dropping it would be a breaking change because all developers will have to add additional imports, so I guess we can leave it for now.

Reply:
Yeah that @_exported inside MapboxSearchController is very convenient to automatically import MapboxSearch. Stay tuned as the language evolves.